### PR TITLE
Adding CompanyName and Country at init

### PR DESCRIPTION
**Pull request by Vaadasch**
Hi,

I create this PR to add INIT_COMPANY_NAME and INIT_COMPANY_COUNTRYCODE in the parameters of the container initialization.

The main interest is to have a working Dolibarr without the need to go to configuration. Automation.
Well, if you want it to be fully configured, you need to, but the bare minimum is country and company name.

I'm not really sure if it is worth it to add these parameters in this image or this way (maybe with a separated file?), but I have added these in order to have a working Dolibarr at container deployment, completing the activation of modules.

I will fully understand if you refuse this PR.

### CODE REVIEW

**Comment by Vaadasch** [comment]
Ah, yes, sorry, I'm not using git on my environment, I only copy files and edit github online with my web browser.

File: README.template [hunk 1]
```diff
@@ -98,6 +98,8 @@ When setup this way, to upgrade version the use of the web interface is mandator
 | **DOLI_ADMIN_PASSWORD**         | *admin*                        | Admin'password
 | **DOLI_URL_ROOT**               | *http://localhost*             | Url root of the Dolibarr installation
 | **DOLI_ENABLE_MODULES**         |                                | Comma-separated list of modules to be activated at install. modUser will always be activated. (Ex: `Societe,Facture,Stock`)
+| **INIT_COMPANY_NAME**           |                                | Set the company name of Dolibarr at container init
+| **INIT_COMPANY_COUNTRYCODE**    |                                | Set the company and Dolibarr country at container init. Need 2-letter codes like "FR", "GB", "US",...
```
Comment (tuxgasy) [inline README.template:102]: Variables must start with **DOLI_**.

**Review by tuxgasy — COMMENTED** [review]
Of course, you can't do a correct PR with GitHub online because you need to run the update.sh script.

I will test later.

File: README.template [hunk 2]
```diff
@@ -98,8 +98,8 @@ When setup this way, to upgrade version the use of the web interface is mandator
 | **DOLI_ADMIN_PASSWORD**         | *admin*                        | Admin'password
 | **DOLI_URL_ROOT**               | *http://localhost*             | Url root of the Dolibarr installation
 | **DOLI_ENABLE_MODULES**         |                                | Comma-separated list of modules to be activated at install. modUser will always be activated. (Ex: `Societe,Facture,Stock`)
-| **INIT_COMPANY_NAME**           |                                | Set the company name of Dolibarr at container init
-| **INIT_COMPANY_COUNTRYCODE**    |                                | Set the company and Dolibarr country at container init. Need 2-letter codes like "FR", "GB", "US",...
+| **DOLI_INIT_COMPANY_NAME**           |                                | Set the company name of Dolibarr at container init
+| **DOLI_INIT_COMPANY_COUNTRYCODE**    |                                | Set the company and Dolibarr country at container init. Need 2-letter codes like "FR", "GB", "US",...
```
Comment (tuxgasy) [inline README.template:102]: No need **INIT** also.

Comment (Vaadasch) [reply README.template]: I was thinking using "INIT" to differentiate between container-configuration and application-configuration, but you're right, that is what "COMPANY_" is for.

File: docker-init.php [hunk 3]
```diff
@@ -24,3 +24,27 @@
...
+    else {
+            printf('Unable to find country '.$countryCode);
+    }
+}
+
+if (!empty(getenv('INIT_COMPANY_NAME'))) {
+    $compname = getenv('INIT_COMPANY_NAME');
+    dolibarr_set_const($db, "MAIN_INFO_SOCIETE_NOM", $compname, 'chaine', 0, '', $conf->entity);
+    $db->commit();
+}
```
Comment (tuxgasy) [inline docker-init.php:50]: You forgot to update env name here before running update.sh

**Review by tuxgasy — CHANGES_REQUESTED** [review]

File: docker-init.php (hunk 3 above)
Comment (Vaadasch) [reply docker-init.php]: My bad, sorry.

**Review by tuxgasy — APPROVED** [review]

**Comment by Vaadasch** [comment]
All green ! :blush:

**Pull request merged**